security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -0,0 +1,56 @@
+[id="cert-manager-operator-release-notes"]
+= {cert-manager-operator} release notes
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-operator-release-notes
+
+toc::[]
+
+TODO: Quick overview
+
+:FeatureName: The {cert-manager-operator}
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+For more information about the {cert-manager-operator}, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
+
+// TODO: Link to upstream release notes too
+
+// TODO: Confirm 1.7.1 version
+[id="cert-manager-operator-release-notes-1.7.1"]
+== Release notes for {cert-manager-operator} 1.7.1
+
+// TODO: Update if needed
+Issued: 2022-03-29
+
+The following advisory is available for the {cert-manager-operator} 1.7.1:
+
+// TODO: Replace
+* link:https://access.redhat.com/errata/RHEA-TODO[RHEA-TODO - TODO]
+
+[id="cert-manager-operator-1.7.1-new-features-and-enhancements"]
+=== New features and enhancements
+
+* TODO
+
+////
+// No bug fixes in initial release
+[id="cert-manager-operator-1.7.1-bug-fixes"]
+=== Bug fixes
+
+* TODO
+////
+
+[id="cert-manager-operator-1.7.1-known-issues"]
+=== Known issues
+
+// TODO: check if this section is needed
+
+* TODO
+
+////
+From slidles:
```

Comment:
@slaskawi Found these two known issues in the TE slides. Are these ones that we want to list in the release notes, and if so, do they have a related BZ or Jira that we can list?

Are there any other known issues that we should add?

Reply:
No, those 2 are the only ones I can think of.

Reply:
@slaskawi do each of these have a Jira or BZ tracking them?

Reply:
Unfortunately not. I'll create them shortly and link back here.

Reply:
BTW, the second point (co-existing multiple operators) is a no-go by design. OLM doesn't allow that.

Reply:
Route support ticket: https://issues.redhat.com/projects/CM/issues/CM-16

Reply:
Thanks @slaskawi! What about "Multiple Cert Manager Operators can not coexist in the same cluster."? Is there a ticket for that (we expect to resolve it at some point)? Or is this an "issue" that is just part of how it is designed (and won't change), and is something that we want to put in a note somewhere when you're installing the operator?

Reply:
Got answer in slack - will note it in the main content and not as a "known issue"